packages/console/src/components/AppContent/components/Sidebar/hook.tsx

```diff
@@ -94,4 +107,6 @@ export const useSidebarMenuItems = (): SidebarSection[] => {
       ],
     },
   ];
+
+  return [sections, findFirstItem(sections)];
```

Comment:
nit: consider using Object instead of Array, so we can get `firstItem` directly with deconstruction